doc/flame_fire_atlas/fire_atlas.md

```diff
@@ -0,0 +1,98 @@
+# FireAtlas
+
+FireAtlas is a tool for handling `texture atlases`. Using FireAtlas we can access images using keys
+from texture atlas like accessing key-value pair.
+
+
+## Texture atlas
+
+A texture atlas(also called a spritesheet or an image sprite in 2d game development) is an image
+that contains data from several smaller images that have been packed together to reduce overall
+dimensions. Texture atlasing enables batching for several static objects that share this texture
+atlas and the same material. Batching reduces the number of draw calls. Fewer draw calls results in
```

Comment:
What do you mean with material here?

Reply:
Also batching doesn't have anything to do with this, so you can leave it out completely.

Reply:
removed that line.